src/utils/mimeUtility.ts

```diff
@@ -90,7 +90,8 @@ export class MimeUtility {
             return false;
         }
 
-        return this.parse(contentTypeString).essence === 'application/javascript';
+        const essence = this.parse(contentTypeString).essence;
+        return essence === 'application/javascript' || essence.includes("text/javascript");
```

Comment:
Why we use `includes` here

Reply:
as it might contain some other text i.e: "Content-Type: text/javascript; charset=utf-8"

Reply:
no need since after the parsing logic, the essence would not include the charset informationion

Reply:
Fixed!